examples/pytorch/logging/pytorch_log_model.ipynb

```diff
@@ -0,0 +1,440 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Run id from the run above: e6f9b749409546758fba3ad9446f71ba\n",
+      "Original model output: tensor([[-0.0688, -0.3263, -0.0642]])\n",
+      "Loaded model output: tensor([[-0.0688, -0.3263, -0.0642]])\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Run id from the run above:\", run.info.run_id)\n",
+    "\n",
+    "# Later, or in a different script, you can load the model using the run ID\n",
+    "loaded_model = mlflow.pytorch.load_model(f\"runs:/{run.info.run_id}/model\")\n",
+    "\n",
+    "# you can now use the loaded model as you would've used the original pytorch model!\n",
+    "loaded_model.eval()\n",
+    "with torch.no_grad():\n",
+    "    sample_input = X_test_tensor[:1]\n",
+    "    loaded_output = loaded_model(sample_input)\n",
+    "    og_output = model(sample_input)\n",
+    "    print(\"Original model output:\", og_output)\n",
+    "    print(\"Loaded model output:\", loaded_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# What you see in the mlflow UI\n",
+    "Please follow [this section of the contributing guide](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.md#javascript-and-ui) to get the UI set up.\n",
+    "NOTE: make sure to launch `mlflow ui` in the same directory that you're running the notebook (where the `mlruns` folder is created as well)\n",
+    "\n",
+    "### Experiment page\n",
+    "Here, you can select the experiment you set in the code above and choose a run to view the model logged during that run. You can also see how your pytorch model has changed in accuracy / loss over different runs in the `Chart` tab.\n",
+    "\n",
+    "<img src=\"./screenshots/experiment.png\" onerror=\"this.onerror=null; this.src='https://i.imgur.com/hiolwEe.png'\" style=\"width: 60%\">\n",
+    "\n",
+    "### Runs detail page\n",
+    "Here, you can see the run ID of this run (used to retrieve the logged model) and the model signature that we set above.\n",
+    "\n",
+    "<img src=\"./screenshots/run.png\" onerror=\"this.onerror=null; this.src='https://i.imgur.com/gJN4f2v.png'\" style=\"width: 60%\">\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  }
+ ],
+ "metadata": {
+  "application/vnd.databricks.v1+notebook": {
+   "dashboards": [],
+   "language": "python",
+   "notebookMetadata": {
+    "pythonIndentUnit": 2
+   },
+   "notebookName": "pytorch log model",
+   "widgets": {}
+  },
+  "kernelspec": {
+   "display_name": "mlflow",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.13"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 0
+}
```

Comment:
Let's delete the screenshots directory, we shouldn't keep things just for future usage.